dirs/dirs.go

```diff
@@ -166,8 +167,8 @@ func SetRootDir(rootdir string) {
 	}
 	GlobalRootDir = rootdir
 
-	switch release.ReleaseInfo.ID {
-	case "fedora", "centos", "rhel", "arch", "manjaro", "lirios":
+	switch {
+	case release.ReleaseInfo.ID == "fedora", strutil.ListContains(release.ReleaseInfo.IDLike, "fedora"), release.ReleaseInfo.ID == "arch", strutil.ListContains(release.ReleaseInfo.IDLike, "arch"):
```

Comment:
I'm not sure if Manjaro uses `/var/lib/snapd/snap` anymore, I think it was reverted back to `/snap` there

Reply:
Manjaro doesn't identify as like Arch, so that should be fine.